internal/tss/services/pool_populator.go

```diff
@@ -85,44 +83,8 @@ func (p *poolPopulator) routeNewTransactions(ctx context.Context) error {
 		if err != nil {
 			return fmt.Errorf("getting latest try for transaction: %w", err)
 		}
-		if try == (store.Try{}) {
-			// there is no try for this transactionm - route to RPC caller channel
+		if try == (store.Try{}) || try.Code == int32(tss.RPCFailCode) || try.Code == int32(tss.NewCode) {
```

Comment:
why are we assigning a tss.NewStatus when try.code is RPCFailCode? In the else-if block below you are sending it to error-jitter channel for the same try.code status

Reply:
Because these transactions have a status of NEW. Sure, I could route them to one of the other channels, but since they are currently in the NEW status, I want to build a payload such that the retries go to the rpc caller channel